orttraining/orttraining/eager/ort_ops.cpp

```diff
@@ -30,8 +30,11 @@ void createInplaceOutputValue(OrtValue& input, V<int64_t> shape, OrtValue* p_mlv
                 element_type, new_shape, p_mlvalue);
 }
 
+template <typename T> 
```

Comment:
was necessary for compilation with clang-12. `std::vector` is not a perfect match since it accepts an additional template argument for the allocator

https://en.cppreference.com/w/cpp/container/vector